src/fetcher/query.ts

```diff
@@ -1,400 +1,549 @@
-// TODO: needs type declaration
-export const idsQuery = (ids: any[]) => {
-  const query = ids.map(
-    (id) => `
-    uuid${id}: uuid(id:${id}) {
-        ... on Entity {
-          alias
-        }
-        ... on Page {
-          alias
-        }
-        ... on TaxonomyTerm {
-          alias
-        }
-      }
-  `
-  )
-  return `{${query}}`
+// Keep this file in sync with the graphQL schema.
+// Maybe automate this one day.
```

Comment:
Haven't used this yet, but https://github.com/apollographql/apollo-tooling seems to be able to generate TypeScript types from a GraphQL query.